html/apps/tweetfeed/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  background-color: gray !important;
+}
+
+md-content.md-default-theme, md-content {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (md-content.md-default-theme) is overqualified, just use .md-default-theme without element name.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2597868539/issues/source?bid=3288751&fileBranchId=3307409#l5)